salt/cache/redis_cache.py

```diff
@@ -275,13 +291,21 @@ def _get_bank_redis_key(bank):
     )
 
 
+def _get_timestamp_key(bank, key):
+    opts = _get_redis_keys_opts()
+    return f"{opts['timestamp_prefix']}{opts['separator']}{bank}/{key}"
```

Comment:
No f strings yet, remember Debian 9
:cry:

Reply:
Bah! :cry:

Reply:
Done... but not happy about it :trollface: